app/Http/Middleware/IsAdmin.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use App\Models\UserType;
+
+class IsAdmin
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle(Request $request, Closure $next)
+    {
+
+        if (UserType::where('name', 'admin')->first()
+            ->users()->where('id', Auth::user()->id)->exists()
+        ) {
```

Comment:
```php
if(Auth::user()->userType()->first()->name == 'admin') {}
```

There may be more than one admin. So this might not work in every case.


PS : Make sure the relation name is correct.